openstudiocore/resources/model/OpenStudio.idd

```diff
@@ -11045,16 +11052,31 @@ OS:HeatPump:WaterToWater:EquationFit:Heating,
    N13, \field Heating Compressor Power Coefficient 4
         \required-field
         \type real
-   N14; \field Heating Compressor Power Coefficient 5
+   N14, \field Heating Compressor Power Coefficient 5
         \required-field
         \type real
+   N15, \field Reference Coefficient of Performance
+        \note This optional field is used to autosize Reference Heating Power Consumption
+        \note COP = Reference Heating Capacity / Reference Heating Power Consumption
+        \type real
+        \minimum> 0.0
+        \default 7.5
+   N16, \field Sizing Factor
```

Comment:
The problem with default values at the end of an object is they show up in that weird gray color in the inspector. I think you should initialize these to the default value in the constructor. I would also prefer them to be a required field and not default in the idd. Both of these changes would require an OS version translation to support existing models. Currently this PR is skirting by with no version translation, but it means old models will always have that IG quirk.